graphql/graphql_test.go

```diff
@@ -452,6 +453,7 @@ func newGQLService(t *testing.T, stack *node.Node, shanghai bool, gspec *core.Ge
 		TrieDirtyCache: 5,
 		TrieTimeout:    60 * time.Minute,
 		SnapshotCache:  5,
+		StateScheme:    rawdb.HashScheme,
```

Comment:
Does the archive mode is needed for GQL test? if not, why still use hashScheme?

Reply:
Yes, GQL test still uses hash scheme and this code is merged from ethereum.